src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -293,13 +294,28 @@ def complete(self, jobs):
                                 logOutput += "\n\n%s\n" % exMsg
 
                             if condorFile == condorLog:
-                                for condorReason in BasicAlgos.findMagicStr(condorFilePath, '<a n="Reason">'):
-                                    logOutput += condorReason
-                                    if "SYSTEM_PERIODIC_REMOVE" in condorReason or "via condor_rm" in condorReason:
-                                        exitCode = 99400
-                                        exitType = "RemovedByGLIDEIN"
+                                # for condor log, search for the information
+                                for matchObj in getIterMatchObjectOnRegexp(condorFilePath, CONDOR_LOG_FILTER_REGEXP):
+                                    condorReason = matchObj.group("Reason")
+                                    if condorReason:
+                                        logOutput += condorReason
+                                        if "SYSTEM_PERIODIC_REMOVE" in condorReason or "via condor_rm" in condorReason:
+                                            exitCode = 99400
+                                            exitType = "RemovedByGLIDEIN"
```

Comment:
Seangchan, I think we should break the loop if we find a 99400 exit code.

Reply:
I am not sure there will be multiple different reasons. I was expecting to have the same reason if there is a duplicate "Reason". Anyway, I will add break statement